policies/queries/ecs/clusters_unused.sql

```diff
@@ -0,0 +1,14 @@
+insert into aws_policy_results
+with attachment as (select distinct cluster_cq_id from aws_ecs_cluster_attachments)
+select :'execution_time'    as execution_time,
+       :'framework'         as framework,
+       :'check_id'          as check_id,
+       'Unused ECS cluster' as title,
+       cluster.account_id,
+       cluster.arn          as resource_id,
+       case
+           when attachment.cluster_cq_id is null then 'fail'
+           else 'pass'
+           end              as status
+from aws_ecs_clusters cluster
+         left join attachment on attachment.cluster_cq_id = cluster.cq_id
```

Comment:
Why does an ECS cluster have to have an attachment?

Reply:
right, that's one more misunderstanding on my end

Reply:
So I think we should remove this check unless we are completely confident about it